docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |set certain contacts as favourites |access to their information easily
+
+|`* * *` |user |select multiple contacts to delete |clean up address book efficiently
+
+|`* * *` |user |save more than one contact number per contact in my address book |account for multiple phone numbers
+
+|`* * *` |organised user |group contacts manually or automatically |access the information I want easily
+
+|`* * *` |user |see my important events on a calendar view |overview of my schedule for the month
+
+|`* * *` |user |share my contact details with my friends |friends do not need to manually add it in
+
+|`* * *` |user |sort the information on the application |view the information in a ordered manner
+
+|`* * *` |user |add information to the application in a intuitive way |not frustrating
```

Comment:
so that i can not frustrating // weird structure. 
but I cannot think of a better word :(

Reply:
Will do.

Reply:
if we are using so that... it can be changes to it is not frustrating

Reply:
Let's keep it as "I can", we've been using that all the while